SimPEG/Directives.py

```diff
@@ -255,7 +255,7 @@ def endIter(self):
             mref = 0
```

Comment:
This function has the exact same name as the one on 208. Which one is the one we want to keep? (first pass they look like duplicates?) @grosenkj